.github/actions/install-eprosima-dependencies-windows/action.yml

```diff
@@ -0,0 +1,36 @@
+
+name: Install eProsima dependencies
+description: Install and setup foonatham-memory, Fast CDR and Fast DDS for linking and building applications in Windows
+inputs:
+  cmake_build_type:
+    description: 'Specifies the build type on single-configuration generators'
+    required: true
+    default: 'Release'
```

Comment:
NIT: Why default if mandatory?

Reply:
I'd keep it as it is in case we miss this input parameter.